Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -25,7 +25,7 @@
 namespace DiscIO
 {
 static u32 ComputeNameSize(const File::FSTEntry& parent_entry);
-static std::string ASCIIToLowercase(std::string str);
+static std::string ASCIIToUppercase(std::string str);
```

Comment:
The hardware test purpose would be to check the filename order.

Windows also has equivalent functions: https://msdn.microsoft.com/en-us/library/chd90w8e.aspx
I won't be surprised if the FS shares properties with Windows knowing that most retailed games were compiled on Windows.